// Copyright (c) 2023 Boston Dynamics, Inc.  All rights reserved.
//
// Downloading, reproducing, distributing or otherwise using the SDK Software
// is subject to the terms and conditions of the Boston Dynamics Software
// Development Kit License (20191101-BDSDK-SL).

syntax = "proto3";

package bosdyn.api;
option go_package = "bosdyn/api/manipulation_api_service";

import "bosdyn/api/manipulation_api.proto";

service ManipulationApiService {
    rpc ManipulationApi(ManipulationApiRequest) returns (ManipulationApiResponse) {}

    rpc ManipulationApiFeedback(ManipulationApiFeedbackRequest)
        returns (ManipulationApiFeedbackResponse) {}

    rpc OverrideGrasp(ApiGraspOverrideRequest) returns (ApiGraspOverrideResponse) {}
}
